pkg/patterns/declarative/pkg/applier/direct.go

```diff
@@ -27,24 +26,36 @@ func (d *DirectApplier) Apply(ctx context.Context,
 	validate bool,
 	extraArgs ...string,
 ) error {
+	tmpFile, err := ioutil.TempFile("", "tmp-manifest")
+	if err != nil {
+		return err
+	}
+	tmpFile.Write([]byte(manifest))
+	tmpFile.Close()
+	tmpFile, err = os.Open(tmpFile.Name())
```

Comment:
We can have an error check for this.

BTW, does this not work if you assign `tmpFile` to `os.Stdin` directly, instead of closing and opening the same file? Both are `*os.File`.